tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.hpp

```diff
@@ -136,7 +136,7 @@ void test_jacobian(
         return map(point, time, functions_of_time);
       };
   // Our default approx value is too stringent for this test
-  Approx local_approx = Approx::custom().epsilon(1e-10).scale(1.0);
+  Approx local_approx = Approx::custom().epsilon(1e-9).scale(1.0);
```

Comment:
Jacobian tests were failing with accuracy of 1e-9 near the outer boundary. Did a test to make sure that the numbers matched with spec this test can be found on Ocean in /home/alexcarpenter/spectre_spec_comparison.cpp the numbers agree to 1e-16 between spec and spectre so it seems safe to increase this tolerance

Reply:
Yes I agree it's safe to increase the tolerance, but only for this test. Your current change will increase it for all domain tests. Instead, in a new commit, edit these functions to take an `approx` as an argument and default it to 1e-10 (the current value). Then for your test, you can pass in a 1e-9 instead.